### PR TITLE
Admin access

### PR DESCRIPTION
**Pull request by gknisely**
We now only process country access logic if the iso country codes match for the begin and end node.  A lot of moving stuff around.

### CODE REVIEW

File: src/mjolnir/graphbuilder.cc [hunk 1]
```diff
@@ -340,13 +341,76 @@ void BuildTileSet(const std::string& ways_file, const std::string& way_nodes_fil
...
     const uint32_t tile_creation_date,
+    const boost::property_tree::ptree& pt,
     std::promise<DataQuality>& result) {
 
   sequence<OSMWay> ways(ways_file, false);
   sequence<OSMWayNode> way_nodes(way_nodes_file, false);
   sequence<Edge> edges(edges_file, false);
   sequence<Node> nodes(nodes_file, false);
 
+  auto database = pt.get_optional<std::string>("admin");
```
Comment (kevinkreiser) [inline src/mjolnir/graphbuilder.cc]: would this chunk of stuff be a good candidate to move to `access.cc` as well?

File: src/mjolnir/countryaccess.cc [hunk 2]
```diff
@@ -50,6 +50,7 @@ uint32_t GetAccess(const uint32_t current_access, const uint32_t country_access,
   return new_access;
 }
 
+// Set the country access for a DE.
 void CountryAccess(DirectedEdge& directededge, const std::vector<int> country_access,
```
Comment (kevinkreiser) [inline src/mjolnir/countryaccess.cc]: it would be could if we'd name this with the word `set` otherwise its very confusing to read in the code

Comment (gknisely) [inline src/mjolnir/countryaccess.cc]: sure np

File: src/mjolnir/graphbuilder.cc (hunk 1 above)
Comment (gknisely) [inline src/mjolnir/graphbuilder.cc]: done

**Comment by kevinkreiser** [comment]
looks good to me :rocket:

**Pull request merged**